x-pack/legacy/plugins/apm/server/lib/transactions/breakdown/index.ts

```diff
@@ -150,7 +152,9 @@ export async function getTransactionBreakdown({
 
   const kpiNames = kpis.map(kpi => kpi.name);
 
-  const timeseriesPerSubtype = resp.aggregations.by_date.buckets.reduce(
+  const timeseriesPerSubtype = (
+    idx(resp.aggregations, _ => _.by_date.buckets) || []
+  ).reduce(
```

Comment:
In other files the `buckets` are extracted, which makes it a little easier on the eyes:
```js
const bucketsByDate = idx(resp.aggregations, _ => _.by_date.buckets) || [];
```